webpack.prod.config.js

```diff
@@ -94,18 +93,25 @@ module.exports = {
         yandex: false,
         windows: false,
       },
+      inject: true,
     }),
     new HtmlWebpackPlugin({
       template: path.join(__dirname, 'src/html/index.html'),
     }),
-    new webpack.DefinePlugin({
-      'process.env': {
-        VERSION: JSON.stringify(`${version}#${build}`),
-        NODE_ENV: JSON.stringify(nodeEnv),
-        GNOSISDB_URL: JSON.stringify(gnosisDbUrl),
-        ETHEREUM_URL: JSON.stringify(ethereumUrl),
-        WHITELIST: JSON.strinfigy(whitelist),
-      },
+    new webpack.EnvironmentPlugin({
+      VERSION: `${version}#${build}`,
```

Comment:
Same here?

Reply:
Nope nope, Environment Plugin just like in Olympia